### PR TITLE
fix(TBD-122838):With tImpalaConnection, hive2 driver is not considere…

### PR DESCRIPTION
**Pull request by cbadillo1603**
…d while running the job and Impala driver is used instead

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet [hunk 1]
```diff
@@ -43,7 +45,7 @@
 	<%@ include file="@{org.talend.designer.components.localprovider}/components/templates/DB/HelpClass/HelpClass.javajet"%>
 	
 
-<%   if(supportImpalaConnector){%>
+<%   if(supportImpalaConnector && "impala".equalsIgnoreCase(impalaDriver)){%>
```
Comment (sponomarova) [inline main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet:48]: Just wondering if this code also handles the same issue in tImpalaInput, for example? 
Is it re-used in other components?

**Review by sponomarova — COMMENTED** [review]
Just wondering if this code also handles the same issue in tImpalaInput, for example?

File: main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet (hunk 1 above)
Comment (cbadillo1603) [reply main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet:48]: Indeed, it is reused in other components like tImpalaCreateTable, tImpalaOutput, tImpalaRow and tImpalaLoad except for tImpalaInput. In fact, tImpalaInput never presented this bug. Others component had it, but this fix is generic for all of them.

Comment (cbadillo1603) [reply main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet:48]: The reason is that such components includes the code in the following manner:

` <%@ include file="@{org.talend.designer.components.bigdata}/components/tImpalaConnection/tImpalaConnection_begin.javajet"%>`

https://github.com/Talend/tbd-studio-se/blob/e5435f465d5b287371a79353ee9ffca935b487d5/main/plugins/org.talend.designer.components.bigdata/components/tImpalaOutput/tImpalaOutput_begin.javajet#L41

Comment (sponomarova) [reply main/plugins/org.talend.designer.components.bigdata/components/tImpalaConnection/tImpalaConnection_begin.javajet:48]: Got it. Thanks for explanation.

**Review by sponomarova — APPROVED** [review]

**Review by lbourgeois — APPROVED** [review]

**Pull request merged**